app/javascript/dashboard/i18n/locale/en/campaign.json

```diff
@@ -0,0 +1,9 @@
+{
+  "CAMPAIGN": {
+    "HEADER": "Campaigns",
+    "HEADER_BTN_TXT": "Create Campaign",
```

Comment:
@sojan-official "Create a campaign" vs "Create Campaign". I suggest the first one.